### ISSUE TITLE
Custom dictionary

### ISSUE THREAD

**Issue by GoogleCodeExporter** [klausw/hackerskeyboard#482]
```
How can new words be added to custom dictionary?


```

Original issue reported on code.google.com by `sixo...@gmail.com` on 14 Jul 2015 at 4:02

**Comment by wxnarwhal** [comment]
You can select the word you typed in the predictive word bar and add it to the dictionary.  Unfortunately, typing any special character clears the word list.  I would love to see this cleared ONLY by a space.  Any consecutive list of characters should be considered a word.  For example, in other keyboards I can add my email address, which is something I type regularly.  However, in Hacker's Keyboard, once I type the @, the predictive list is cleared out.  I love this keyboard despite this, and it's the only keyboard I ever use on Android.  Many thanks for the great work.

**Comment by markwr** [comment]
Yes, I most heartily agree!

**Comment by shshamdani** [comment]
I agree with this, too. Adding a whole sequence of characters to the custom dictionary would be great. E.g. words with hyphens in the middle, words like rock'n'roll, etc. ;P
